.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+<!--
+    Please sign your commit and our DCO.
+
+    Details on how to sign commits can be found at https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits
```

Comment:
I think we should not require GPG signing of git commits. That sets the bar too high.

Reply:
Heh, you beat me to it ;-)

Reply:
Due to they way Git works, `git --sign` is valid and results in `--signoff`, which is where this is coming from. But yes, you are correct, thanks :)